prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My Guitars")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: "))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add} added.")
+        # guitars.append(Guitar(name, year, cost))
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
```

Comment:
May find it useful to have a comment above these saying they are part of the program for testing?